wfe/context.go

```diff
@@ -0,0 +1,85 @@
+package wfe
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/core"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+type requestEvent struct {
+	ID           string    `json:",omitempty"`
+	RealIP       string    `json:",omitempty"`
```

Comment:
Whose real IP?  And "real" as opposed to what?  Do you mean something like `ServerPrivateIP` or `ServerPublicIP`?